Pod/Classes/PhxChannel.h

```diff
@@ -26,12 +27,12 @@
 @property (nonatomic, weak) id<PhxChannelDelegate> delegate;
 @property (nonatomic, weak) PhxSocket* socket;
 @property (nonatomic, readonly) ChannelState state;
-@property (nonatomic, retain) NSString* topic;
-@property (nonatomic, retain) NSDictionary *params;
+@property (nonatomic) NSString* topic;
+@property (nonatomic) NSDictionary *params;
```

Comment:
would use `strong` here, assuming the original author wanted to make the memory-management explicit here.